SU2_CFD/include/fluid/CCoolProp.hpp

```diff
@@ -41,7 +41,7 @@ namespace CoolProp {
  * \brief Child class for defining fluid model from CoolProp library.
  * \author: P.Yan
  */
-class CCoolProp final : public CFluidModel {
+class CCoolProp final : public CFluidModel{
```

Comment:
```suggestion
class CCoolProp final : public CFluidModel {
```

Reply:
changed

Reply:
@PENGYAN777 better to use "commit suggestion" directly, especially for these small things, then it is immediately clear that it was implemented exactly as was suggested.

Reply:
I will try this, thank you.